iTechArt.Service/Parsers/GroceryParsers.cs

```diff
@@ -74,13 +93,22 @@ public async Task RecordExcelToDatabase(IFormFile formFile)
                     {
                         var sheet = p.Workbook.Worksheets.First();
                         var count = sheet.Dimension.Rows;
-
-                        for (int i = 2; i < count; i++)
+                        
+                        for (int index = 2; index < count; index++)
                         {
-                            _grocery.Add(GroceryMapper.XlsxMapper(sheet, i));
-
+                            var groceryDTO = new GroceryDTO
+                            {
+                                FirstName = sheet.Cells[index, 1].Value?.ToString(),
```

Comment:
We described on the last lesson, that it would be better to use sheet.GetValue<>().....